UPGRADE-4.2.md

```diff
@@ -55,6 +55,7 @@ Finder
 Form
 ----
 
+ * Using the `format` option of the `DateType` and `DateTimeType` when the `html5` option is enabled is deprecated.
```

Comment:
2nd "the" is strange when reading the sentence, isn't it? might miss a word, or should be removed? (same below)